app/src/main/java/com/dgaf/happyhour/View/ViewPagerFragment.java

```diff
@@ -51,7 +62,7 @@ public Fragment getItem(int position) {
             // getItem is called to instantiate the fragment for the given page.
             // Return a PlaceholderFragment (defined as a static inner class below).
 
-            switch(position) {
+            switch (position) {
                 case 0: {
                     return mDrinkFragment;
```

Comment:
I think you can change this case statements to constants like case MONDAY: or case POSITION: like you had before.